src/main/java/backend/resource/TurboLabel.java

```diff
@@ -95,6 +95,17 @@ public static final TurboLabel getFirstMatchingTurboLabel(List<TurboLabel> label
         return firstMatchingLabel.get();
     }
 
+    /**
+     * @param allLabels
+     * @param names
+     * @return list of labels in allLabels that matches list of names
+     */
+    public static List<TurboLabel> generateLabels(List<TurboLabel> allLabels, List<String> names) {
```

Comment:
The method name could be a bit more specific. 'Generate' implies something is being created, but this finds the matching TurboLabels in a list.

Also, can this share code with getMatchedLabels?